datasketch/lshforest.py

```diff
@@ -128,6 +129,29 @@ def query(self, minhash: MinHash, k: int) -> List[Hashable]:
             r -= 1
         return list(results)
 
+    def get_minhash_from_key(self, key: Hashable) -> MinHash:
+        """
+        Returns the MinHash value that corresponds to the given key in the LSHForest,
+        if it exists. This is useful for when we want to manually check the 
+        Jaccard Similarity for the top-k results from a query.
+
+        Args:
+            key (Hashable): The key whose MinHash we want to retrieve.
+
+        Returns:
+            MinHash: The corresponding MinHash value for the provided key.
+        """
+        byteslist = self.keys.get(key, None)
+        if byteslist is None:
+            raise KeyError(f"The provided key does not exist in the LSHForest: {key}")
+        hashvalues = np.array([], dtype=np.uint64)
+        for item in byteslist:
+            # unswap the bytes, as their representation is flipped during storage
+            hv_segment = np.frombuffer(item, dtype=np.uint64).byteswap()
+            hashvalues = np.append(hashvalues, hv_segment)
+        minhash = MinHash(hashvalues=hashvalues)
```

Comment:
The original MinHash might be using different hash functions and different seed, so I think the appropriate return value for this function is the hash values. 

The name should be `get_minhash_hashvalues`.

Reply:
Ah, you're right, I've amended the PR with the latest commit. Please let me know if there are any other changes